Source/SwiftLintFramework/Extensions/NSFileManager+SwiftLint.swift

```diff
@@ -21,7 +21,7 @@ extension FileManager: LintableFileManager {
             .standardizingPath
 
         // if path is a file, it won't be returned in `enumerator(atPath:)`
-        if absolutePath.bridge().isSwiftFile() && absolutePath.isFile {
+        if absolutePath.isFile {
```

Comment:
I'm kinda worried that if we extend `included`/`excluded` to support globbing, this change would attempt to lint non-Swift files if `Sources/**` were specified, for example.